lib/twitter_cldr/js/tasks/tasks.rb

```diff
@@ -84,6 +75,22 @@ def build(options = {})
           )
         end
 
+        def write_bundle(file_path, file_contents)
+          out_file = file_path
+
+          if file_path.is_a?(Array)
+            out_file = File.join(file_path)
+          end
+
+          write_file(out_file, file_contents)
+        end
+
+        def write_file(file_name, file_contents)
+          File.open(file_name, "w+") do |f|
```

Comment:
This is how the writes were being done earlier. I just assumed that was the de facto way of doing it in Ruby! My bad. I should have looked up further.